### ISSUE TITLE
VS Code Web: Silence Curl

### ISSUE THREAD

**Issue by djarbz** [coder/modules#194]
If these lines had the options `-Ss` then we would not see the extraneous curl output in the log.
https://github.com/coder/modules/blob/7e3743739edb0861fa6b569126066de4fc38af7f/vscode-web/run.sh#L22-L23

```
Installing Microsoft Visual Studio Code Server!
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

Checking if dotfiles repository already exists...
Found dotfiles repository at /home/coder/.config/coderv2/dotfiles

100  6279  100  6279    0     0  16266      0 --:--:-- --:--:-- --:--:-- 16266
```

**Comment by michaelbrewer** [comment]
I have a related change that fixes this too

**Comment by michaelbrewer** [comment]
@matifali  - this should be resolved now with the merge of #195
